render/pod.go

```diff
@@ -57,6 +57,34 @@ var PodServiceRenderer = ConditionalRenderer(renderKubernetesTopologies,
 	)),
 )
 
+// DeploymentRenderer is a Renderer which produces a renderable kubernetes deployments
+// graph by merging the pods graph and the deployments topology.
+var DeploymentRenderer = ApplyDecorators(
+	FilterEmpty(report.Pod,
+		MakeReduce(
+			MakeMap(
+				MapPod2Deployment,
+				PodRenderer,
+			),
+			SelectDeployment,
+		),
+	),
```

Comment:
Is this right?  I was under the impression is was pod -> replica set -> deployment?